### PR TITLE
Add Python 3.6 test environment

### PR DESCRIPTION
**Pull request by ingwinlu**
See commits.

### CODE REVIEW

**Comment by ingwinlu** [comment]
@getpelican/reviewers go for review :)

File: docs/contribute.rst [hunk 1]
```diff
@@ -47,13 +47,16 @@ Or using ``pip``::
 
     $ pip install -e .
 
+To conveniently test vs multiple python versions we also provide a tox file.
```
Comment (justinmayer) [inline docs/contribute.rst]: > To conveniently test on multiple Python versions, we also provide a ``.tox`` file.

File: .travis.yml [hunk 2]
```diff
@@ -7,19 +7,20 @@ env:
...
+  - TOX_ENV=py35
 matrix:
-  include:
-    - python: 3.5
-      env:
-      - TOX_ENV=py35
+    include:
+        - python: 3.6
+          env:
+              - TOX_ENV=py36
```
Comment (iKevinY) [inline .travis.yml]: It would probably be better to keep indentation here consistent with the rest of the file (2 spaces).

File: tox.ini [hunk 3]
```diff
@@ -37,8 +37,7 @@ import-order-style = cryptography
 [testenv:flake8]
 basepython = python2.7
 deps =
-    flake8 <= 2.4.1
-    git+https://github.com/public/flake8-import-order@2ac7052a4e02b4a8a0125a106d87465a3b9fd688
```
Comment (iKevinY) [inline tox.ini]: Just confirming: we don't need to pin flake8-import-order to this commit anymore?

Comment (ingwinlu) [inline tox.ini]: yes

Back when i added the environment with the flake8 check there was a bug with one of the rules which was fixed in the repo, but not in the release (something with import ordering if i remember  right).

**Comment by ingwinlu** [comment]
Updated with review-feedback

File: .travis.yml [hunk 4]
```diff
@@ -7,19 +7,20 @@ env:
...
+  - TOX_ENV=py35
 matrix:
-  include:
-    - python: 3.5
-      env:
-      - TOX_ENV=py35
+    include:
+        - python: 3.6
+          env:
+            - TOX_ENV=py36
```
Comment (iKevinY) [inline .travis.yml]: There's still some indentation that has snuck in here 😛 (basically just remove the spaces that show up in the diff)

Comment (ingwinlu) [inline .travis.yml]: Ah the good old 'let me just commit this real quick before bed' strikes again...

**Comment by justinmayer** [comment]
Many thanks to @ingwinlu for the enhancements and to @iKevinY for the review.  😃

**Pull request merged**